Python/Python.sublime-syntax

```diff
@@ -1004,34 +1014,21 @@ contexts:
         1: punctuation.section.mapping.begin.python
         2: punctuation.section.mapping.end.python
       push: after-expression
-    - match: \{(?={{simple_expression}}:|\s*\*\*)
-      scope: punctuation.section.mapping.begin.python
-      push: inside-dictionary
-    - match: \{(?={{simple_expression}}[,}]|\s*\*)
+    - match: (?=\{)
+      branch_point: dictionaries-and-sets
+      branch:
+        - try-set
+        - try-dictionary
+
+  try-set:
```

Comment:
- `try-...` feels like a try-catch statement related context.
- Naming of branch contexts should be consistent.